pytential/symbolic/mappers.py

```diff
@@ -137,6 +139,45 @@ def map_interpolation(self, expr):
 
         return type(expr)(expr.from_dd, expr.to_dd, operand)
 
+# }}}
+
+
+# {{{ IntGTermCollector
+
+class IntGTermCollector(IdentityMapper):
```

Comment:
Add docstrings, add to docs?

Reply:
Added in dfcee25. Let me know what you think!

The mappers aren't actually hooked into the docs, so I didn't add them anywhere. Should they be?

Reply:
Once they're in a module of direct-solver-specific mappers (see below) (that's perhaps hooked into the docs in a place marked "internal API only"), I don't see why not.

Reply:
Yep, done in d124100.